paymentsheet/src/main/java/com/stripe/android/link/LinkController.kt

```diff
@@ -60,6 +60,27 @@ class LinkController @Inject internal constructor(
         )
     }
 
+    /**
+     * [CRYPTO ON-RAMP ONLY] Present the Link authentication screen.
+     *
+     * This will launch the Link activity where users can authenticate with their Link account.
+     * The authentication flow will close after successful authentication instead of continuing
+     * to payment selection. The result will be communicated through the [PresentForAuthenticationCallback]
+     * provided during controller creation.
+     *
+     * If a presentation is already in progress, this call will be ignored.
+     *
+     * @param email The email address to use for Link account lookup. If provided and the email
+     * matches an existing Link account, the user will be able to authenticate with that account.
+     * If null, the user will need to sign in or create a Link account.
+     */
+    fun presentForAuthentication(email: String?) {
```

Comment:
I know the class is already restricted, but out of abundance of caution can you restrict this function specifically since we intend for it to be internal for the foreseeable future?
```suggestion
    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
    fun presentForAuthentication(email: String?) {
```